packages/babel-plugin/src/css-prop/__tests__/object-literal.test.tsx

```diff
@@ -729,4 +729,18 @@ describe('css prop object literal', () => {
 
     expect(actual).toInclude('{color:red}');
   });
+
+  it('should retain keys for mapped react components', () => {
```

Comment:
You should add one test for styled components like you have for the css prop here

Reply:
There's no key added to the generated react components because it's not needed